Server/src/config/config.xml

```diff
@@ -108,7 +108,7 @@
                         response_format="xml"/>
       <commandProcessor classname="org.openas2.cmd.processor.RestCommandProcessor" 
                         enabled="$properties.restapi.command.processor.enabled$"
-                        baseuri="http://localhost:8443"
```

Comment:
Hi Brian,
   This configuration is used not only on Docker Images, but also on Standalone servers. Therefore for security reasons I lean towards having a more secure by default configuration limiting the exposure of the API to localhost and SSL.

   An alternative solution for running the WebUI as a docker image could be to either provide more documentation related to the configuration changes needed or to include a sample config.xml file.

Reply:
Makes since. I've pushed a new commit to change it back to localhost, and I've adjusted the WebUI docker instructions instead.

Reply:
> Makes sense. I've pushed a new commit to change it back to localhost, and I've adjusted the WebUI docker instructions instead.

Thanks for the update,
Now, I've setup new openas2 server version 3.6.0 with webUI and created some Partners configuration and connection for testing but i am not able to see the data location of the Partners, connections, certificates, send and received folder path inside of the amazon linux.
Could you please help me to where i can found the path of the data folders inside in Linux machine.

Reply:
Sorry, I can't help with that. I'd recommend you create a new discussion to get help from others. https://github.com/OpenAS2/OpenAs2App/discussions